terraform/modules/elasticsearch/main.tf

```diff
@@ -27,3 +27,24 @@ resource "aws_elasticsearch_domain" "main" {
     aws_iam_service_linked_role.es,
   ]
 }
+
```

Comment:
I guess this was an intentional removal as part of this PR?

Reply:
Soz - for some reason I saw this as being removed, rather than just not added! D'oh!

Reply:
Ha no problem - removed it now anyway, save some confusion until we understand it.  Can always follow up with a second change.